replica/database.hh

```diff
@@ -1288,6 +1288,15 @@ public:
         }
     };
 
+    using ks_cf_t = std::pair<sstring, sstring>;
+    using ks_cf_to_uuid_t =
+        flat_hash_map<ks_cf_t, table_id, utils::tuple_hash, string_pair_eq>;
+    class column_families_data {
```

Comment:
1. It's contains metadata, not data
2. it can be modernized in this opportunity
So how about `tables_metadata` and a respective `get_tables_metadata()` method?